src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -2469,6 +2469,9 @@ namespace SurfaceGeometry {
                                                   "Unable to apportion the zone floor area. Space Floor Areas are zero.");
                             }
                         }
+                    } else {
+                        if (state.dataHeatBal->Zone(ZoneNum).CalcFloorArea > 0.0)
+                            state.dataHeatBal->Zone(ZoneNum).FloorArea = state.dataHeatBal->Zone(ZoneNum).CalcFloorArea;
```

Comment:
Set Zone FloorArea = Zone CalcFlorrArea. Otherwise, the program is crashed with debug due to ZoneFloorArea = 0.

Reply:
I wonder why we need a separate CalcFloorArea member variable then.  I won't drag out this tiny fix, but I would have expected `FloorArea` to be assigned zero (or `DataSizing::AutoCalculate`), and then calculated at the right time if it needs to be.  Oh well, this is fine.